docs/getting-started.rst

```diff
@@ -174,117 +158,71 @@ Note that if one of these columns exists, it will be overwritten.
 +-------------------+-----------------------------------------------------+------------+
-|wohngeld_basis     |                                                     | Float      |
+|abgst              |Monthly Capital Income Tax Due, individual           | Float      |
 +-------------------+-----------------------------------------------------+------------+
-|wohngeld_basis_hh  |                                                     | Float      |
+|soli               |Monthly Solidarity Surcharge due, individual         | Float      |
 +-------------------+-----------------------------------------------------+------------+
-|ar_alg2_ek_hh      |                                                     | Float      |
+|soli_tu            |Monthly Solidarity Surcharge due, couple sum         | Float      |
 +-------------------+-----------------------------------------------------+------------+
-|alg2_grossek_hh    |                                                     | Float      |
+|kindergeld         |Monthly Child Benefit, individual to the child       | Float      |
 +-------------------+-----------------------------------------------------+------------+
-|mehrbed            |                                                     | Float      |
+|kindergeld_tu      |Monthly Child Benefit, household sum                 | Float      |
 +-------------------+-----------------------------------------------------+------------+
-|regelbedarf        |                                                     | Float      |
+|incometax          |Monthly Income Tax Due, individual                   | Float      |
 +-------------------+-----------------------------------------------------+------------+
-|regelsatz          |                                                     | Float      |
+|incometax_tu       |Monthly Income Tax Due, couple sum                   | Float      |
 +-------------------+-----------------------------------------------------+------------+
-|alg2_kdu           |                                                     | Float      |
+|uhv                |Alimony Advance payment, individual to the child     | Float      |
 +-------------------+-----------------------------------------------------+------------+
-|uhv_hh             |                                                     | Float      |
+|regelbedarf        |Household socio-economic *need*, incl. housing cost  | Float      |
 +-------------------+-----------------------------------------------------+------------+
-|ekanrefrei         |                                                     | Float      |
+|regelsatz          |Household socio-economic *need*, lump-sum            | Float      |
 +-------------------+-----------------------------------------------------+------------+
-|kiz_temp           |                                                     | Float      |
+|alg2_kdu           |Household Appropriate Housing Cost                   | Float      |
```

Comment:
??? Please look for a better translation, this sounds strange.

Reply:
Changed to "Appropriate housing costs for a family in *need*"